main.go

```diff
@@ -188,10 +188,18 @@ func main() {
 	log.Infof("created kafka producer: %s", producer.String())
 	go producer.StartProducer(kafkaProducerMessagesChan)
 
+	DB, err := db.OpenDB(*cfg)
```

Comment:
Nit-picky:  i would make the `DB` lower case so its clear that we are not trying to export the `DB` variable.